Microsoft.NET.Build.Containers.IntegrationTests/TargetsTests.cs

```diff
@@ -121,4 +121,28 @@ public void ShouldNotIncludeSourceControlLabelsUnlessUserOptsIn(bool includeSour
         };
     }
 
+    [InlineData("7.0.100", "7.0", "7.0")]
+    [InlineData("7.0.100-preview.7", "7.0", "7.0")]
+    [InlineData("7.0.100-rc.1", "7.0", "7.0")]
+    [InlineData("8.0.100", "8.0", "8.0")]
+    [InlineData("8.0.100", "7.0", "7.0")]
+    [InlineData("8.0.100-preview.7", "8.0", "8.0-preview.7")]
+    [InlineData("8.0.100-rc.1", "8.0", "8.0-rc.1")]
+    [InlineData("8.0.100-rc.1", "7.0", "7.0")]
+    [InlineData("6.0.100", "6.0", "6.0")]
+    [InlineData("6.0.100-preview.1", "6.0", "6.0")]
```

Comment:
should we add some tests for VS releases: 7.0.200, 8.0.200, 8.0.200-preview3?

Reply:
Done, but the SDK feature band number has no impact on tag calculation per the new versioning spec - it's all derived from the major/minor and preview status of the SDK version number.

Reply:
Of course you were right :D Adding a 8.0.200-preview.3 case exposed a bug in the generation logic where the preview status doesn't matter after the 100-level release.